lightning/src/ln/channel.rs

```diff
@@ -4878,10 +4878,6 @@ impl<SP: Deref> ChannelContext<SP> where SP::Target: SignerProvider {
 		}
 
 		let funding_tx_confirmations = height as i64 - funding.funding_tx_confirmation_height as i64 + 1;
-		if funding_tx_confirmations <= 0 {
-			funding.funding_tx_confirmation_height = 0;
-		}
```

Comment:
@wpaulino Could you check my understanding on this?

When unconfirming a funding transaction, the passed `height` will be the previous `funding_tx_confirmation_height - 1` while `funding.funding_tx_confirmation_height` is now zero.
- For non-zero conf, the previous check should result in returning `false` before hitting this.
- But for zero-conf, if it had been confirmed and then is unconfirmed, the next check will not be true resulting in returning `true` on the last line.

Does that seem right?

Reply:
Renamed to `check_funding_meets_minimum_depth` and made a separate commit to special case 0-conf.